libglusterfs/src/iobuf.c

```diff
@@ -23,8 +23,9 @@
 /* Make sure this array is sorted based on pagesize */
 static const struct iobuf_init_config gf_iobuf_init_config[] = {
     /* { pagesize, num_pages }, */
-    {128, 1024},     {512, 512},       {2 * 1024, 512}, {8 * 1024, 128},
-    {32 * 1024, 64}, {128 * 1024, 32}, {256 * 1024, 8}, {1 * 1024 * 1024, 2},
+    {132 * 1024, 16},
```

Comment:
I'm pretty sure that on one hand, 16 are not enough, OTOH, you should have created a pool of 128K as well (which is the default size). I'd go with 64 of 128K and 64 of 132. What I'm unsure is if 132 is enough, or we need 128K+4K+few more bytes...